src/config.py

```diff
@@ -0,0 +1,387 @@
+import os
+import json
+import base64
+
+
+class Config:
```

Comment:
I don't know enough Python to say if this class should go in the __init__ file instead of its own file.  @NickAnderegg?